ci-operator/step-registry/cert-manager/upgrade/cert-manager-upgrade-ref.yaml

```diff
@@ -0,0 +1,28 @@
+ref:
+  as: cert-manager-upgrade
+  from_image:
+    namespace: ci
+    name: verification-tests
+    tag: latest
+  grace_period: 20m
+  commands: cert-manager-upgrade-commands.sh
+  cli: latest
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 300m
+      memory: 200Mi
+  env:
+    - name: UPGRADE_CATSRC
+      documentation: The name of the catalogsource to be used for the operator upgrade.
+      default: "qe-app-registry"
+    - name: UPGRADE_CHANNEL
+      documentation: The name of the subscription channel that contains the UPGRADE_CSV.
+      default: "stable-v1"
+    - name: UPGRADE_CSV
+      documentation: The name of the specific CSV to upgrade to. Automatic upgrade is enabled by default.
+      default: ""
```

Comment:
Does "Automatic upgrade is enabled by default" relate to this env var's description?
And what does `default: ""` mean?
Does "env:" need specify this env var?
Ref: https://github.com/openshift/release/pull/55016#discussion_r1701618000

Reply:
I will reword the description.

```

Reply:
"env:" should not specify this env var. Let script detect if users specified ss previous comment

Reply:
I'd keep the `default: ""` for UPGRADE_CSV, then we can detect if users specify them by checking if `${UPGRADE_CSV}` is empty. In this way, https://github.com/openshift/release/pull/55016#discussion_r1701726427 won't be a problem anymore.